reporting/api/src/main/java/org/jboss/windup/reporting/model/ApplicationReportModel.java

```diff
@@ -78,6 +79,22 @@
      */
     @Property(DISPLAY_IN_APPLICATION_LIST)
     void setDisplayInApplicationList(Boolean displayInApplicationList);
+    
+
+    /**
+     * Indicates whether the Report is index of All Application 
+     * @return
+     */
+    @Property(GLOBAL_APPLICATION_LIST_REPORT)
+    Boolean getGlobalApplicationListReport();
```

Comment:
Is this a leftover from a previous idea? It doesn't really look like the value of this property is read anywhere?